### PR TITLE
Adds multi-z support for pinpointers [NSV13 port]

### PR DESCRIPTION
**Pull request by covertcorvid**
## About The Pull Request
Adds "above" and "below" icons to pinpointers and code for using them

## Why It's Good For The Game
Helps make multi-z more viable

## Changelog
:cl:
add: Added multi-z support for pinpointers
imageadd: Added above and below overlays for pinpointers
/:cl:

### CODE REVIEW

File: code/game/objects/items/pinpointer.dm [hunk 1]
```diff
@@ -67,9 +67,14 @@
 		return
 	var/turf/here = get_turf(src)
 	var/turf/there = get_turf(target)
+
 	if(here.z != there.z)
-		add_overlay("pinon[alert ? "alert" : ""]null[icon_suffix]")
-		return
+		if(here.z != there.z)
```
Comment (PowerfulBacon) [inline code/game/objects/items/pinpointer.dm]:
```
if(here.z != there.z)
	if(here.z != there.z)
         if(here.z > there.z)
				add_overlay("pinon_below[icon_suffix]")
			else
				add_overlay("pinon_above[icon_suffix]")
			return
```

**Review by PowerfulBacon — CHANGES_REQUESTED** [review]
Remove double z level check

**Review by PowerfulBacon — APPROVED** [review]

**Comment by ike709** [comment]
Looks like there's a stray pixel in the top left corner in part of the animation.

**Comment by covertcorvid** [comment]
> 
> 
> Looks like there's a stray pixel in the top left corner in part of the animation.

It looks okay on my screen. Any chance you can get a screenshot?

**Comment by ike709** [comment]
https://github.com/BeeStation/BeeStation-Hornet/pull/1874/checks?check_run_id=707453872

Watch the icondiffbot's preview of the sprites.

**Comment by covertcorvid** [comment]
I don't see it still
https://streamable.com/lfxdid

**Comment by ike709** [comment]
https://streamable.com/zvsnv8

Because you're using lightmode.

**Comment by covertcorvid** [comment]
Just deleting the area got me no changes, so I painted the transparent part electric green and then deleted it all again, but the icondiffbot preview still shows it. I'm not sure what to do about that, I'm open to suggestions.
I can also record what it looks like in game in an hour or so as evidence.

**Comment by Jalleo** [comment]
I have also had a look and I cannot find the pixel via the DMI itself. I believe there might be corruption in the icon bot itself when it tried to animate it.

**Comment by ike709** [comment]
Eh, maybe it's just icondiffbot weirdness.

**Pull request merged**